src/Components/WebAssembly/WebAssembly.Authentication/src/NavigationManagerExtensions.cs

```diff
@@ -30,14 +30,14 @@ public static class NavigationManagerExtensions
     /// <param name="manager">The <see cref="NavigationManager"/>.</param>
     /// <param name="logoutPath">The path to navigate too.</param>
     /// <param name="returnUrl">The url to redirect the user to after logging out.</param>
-    public static void NavigateToLogout(this NavigationManager manager, string logoutPath, string returnUrl)
+    public static void NavigateToLogout(this NavigationManager manager, string logoutPath, string? returnUrl)
     {
         manager.NavigateTo(logoutPath, new NavigationOptions
         {
             HistoryEntryState = new InteractiveRequestOptions
             {
                 Interaction = InteractionType.SignOut,
-                ReturnUrl = returnUrl
+                ReturnUrl = returnUrl ?? manager.Uri
```

Comment:
I'm not sure about this change. Can `InteractiveRequestOptions.ReturnUrl` be null?

Reply:
It seems like we assume the return URL can be `null` elsewhere in `WebAssembly.Authentication`, e.g. [`RemoteAuthenticatorViewCore`](https://github.com/dotnet/aspnetcore/blob/a80400004a4b2e5c0d3f9cff67641d0a47b0202c/src/Components/WebAssembly/WebAssembly.Authentication/src/RemoteAuthenticatorViewCore.cs#L370). Maybe we can make `InteractiveRequestOptions.ReturnUrl` nullable and not required?

cc @javiercn